rules_php_gapic/php_gapic.bzl

```diff
@@ -70,9 +71,13 @@ def php_gapic_srcjar(
     if transport != "grpc+rest" and transport != "rest":
         fail("Error: Only 'grpc+rest' or 'rest' transports are supported")
 
+
     # Set plugin arguments.
     plugin_args = ["metadata"]  # Generate the gapic_metadata.json file.
     plugin_args.append("transport=%s" % transport)
+    
+    if rest_numeric_enums: # Generate REGAPIC param for enum response encoding.
```

Comment:
nit: "requesting response enums be JSON-encoded as numbers, not strings"

Reply:
Done.